azure/pipelines/dotnet/ci.yml

```diff
@@ -168,13 +172,31 @@ steps:
           &'C:\Program Files\dotnet\dotnet.exe' msbuild /target:EndPlatinaTestInit '${{ parameters.Solution }}'
           # Ignore msbuild errors because targets might not be present in projects.
           exit 0;
-  - task: DotNetCoreCLI@2
-    displayName: 'DotNet Test'
-    inputs:
-      command: 'test'
-      projects: '**/*[Tt]ests/*.csproj'
-      arguments: '--configuration ${{ parameters.BuildConfiguration }} --no-restore --no-build'
-    condition: succeededOrFailed()
+
+  - ${{ if ne(parameters.azureSubscription, '') }}:
```

Comment:
I suggest you put the 'regular' testing method first: condition: ${{ if eq(parameters.azureSubscription, '') }}